include/odp/api/spec/packet_types.h

```diff
@@ -304,6 +304,15 @@ typedef struct odp_packet_reass_partial_state_t {
 	uint64_t elapsed_time;
 } odp_packet_reass_partial_state_t;
 
+/**
+ * Information about a stripped VLAN header
+ */
+typedef struct odp_packet_vlan_info_t {
+	/** VLAN ID in host byte order */
+	uint16_t vlan_id;
+
+} odp_packet_vlan_info_t;
```

Comment:
Should PCP and DEI be here as well?

Reply:
They could be and the reason for having the structure was to be able to add those later in a basically backward compatible manner. I just did not include them yet. I wonder if I should.

One more thing that could perhaps be included at some point is the TPID (the ethertype of the stripped VLAN tag) which would become useful if possibility to strip other VLAN tag types (QinQ) gets added in the API.

Reply:
From client POV, if this is the only way to get information about a stripped VLAN tag (directly, I guess one could infer some of these from e.g. classification results), it would make sense to add also the rest of the data.